src/main.rs

```diff
@@ -50,24 +53,42 @@ fn get_signer(
 fn get_owners_and_mints(
     sub_matches: &ArgMatches<'_>,
     allow_null_signer: bool,
+    rpc_client: &RpcClient,
     wallet_manager: &mut Option<Arc<RemoteWalletManager>>,
 ) -> (Vec<Box<dyn Signer>>, Option<Vec<Pubkey>>) {
     let mints = if sub_matches.is_present("mint") {
-        Some(
-            sub_matches
-                .values_of("mint")
-                .unwrap()
-                .map(|p| {
-                    get_signer(
-                        sub_matches,
-                        p,
-                        wallet_manager,
-                        /* allow_null_signer = */ true,
+        let mints = sub_matches
+            .values_of("mint")
+            .unwrap()
+            .map(|p| {
+                get_signer(
+                    sub_matches,
+                    p,
+                    wallet_manager,
+                    /* allow_null_signer = */ true,
+                )
+                .pubkey()
+            })
+            .collect::<Vec<_>>();
+        for mint in &mints {
+            let mint_account = rpc_client.get_account(mint).unwrap_or_else(|_|
+                panic!(
+                    "Account {} expected to be an SPL token mint, but does not exist. Maybe this is a system account?",
+                    mint
                     )
-                    .pubkey()
-                })
-                .collect::<Vec<_>>(),
-        )
+                );
```

Comment:
odd indent?

Reply:
it was very strange, i changed this a few different ways, and cargo fmt did nothing with it... let me see if i can get it like other one